build/commands/lib/test.js

```diff
@@ -145,6 +169,15 @@ const runTests = (passthroughArgs, suite, buildConfig, options) => {
   }
 
   if (options.output) {
+    // Clear any previous output
+    const outputPath = path.join(config.srcDir, options.output)
+    if (!deleteFile(outputPath)) {
+      return false
+    }
+    // Also suite results file.
+    if (!deleteFile(allResultsFilePath)) {
+      return false
```

Comment:
these returns do nothing here. The error will be logged, but `npm run test` won't fail.

Consider removing all try/catch from `deleteFile`/`appendToResultsFile` and just let them throw. No need to handle true/false from these functions at all.